src/NLog/LayoutRenderers/ExceptionLayoutRenderer.cs

```diff
@@ -176,8 +224,8 @@ protected override void Append(StringBuilder builder, LogEventInfo logEvent)
         /// Appends the Message of an Exception to the specified <see cref="StringBuilder" />.
         /// </summary>
         /// <param name="sb">The <see cref="StringBuilder"/> to append the rendered data to.</param>
-        /// <param name="ex">The exception containing the Message to append.</param>
-        protected virtual void AppendMessage(StringBuilder sb, Exception ex)
+        /// <param name="ex">The exception containing the Message to append.</param>        
+        private static void AppendMessage(StringBuilder sb, Exception ex)
```

Comment:
I think 2 won't work with virtual? Action and virtual won't work together? (method override in sub class, maybe unit test it?)  If it does, I prefer 2 and otherwise we should do 1.